Jenkinsfile.release

```diff
@@ -0,0 +1,87 @@
+#!groovy
+
+def retagImage(String image_url, String source_tag, String dest_tag) {
+    // Pulls down image with source_tag and re-tags with dest_tag
+    // before pushing back to repository
+
+    sh "docker pull ${image_url}:${source_tag}"
+    sh "docker tag  ${image_url}:${source_tag} ${image_url}:${dest_tag}"
+    sh "docker push ${image_url}:${dest_tag}"
+}
+
+node{
+   properties(
+       [parameters(
+           [string(defaultValue: 'develop', description: 'SHA of the commit you wish to deploy.', name: 'GIT_COMMIT', trim: false),
```

Comment:
Since this isn't the full `SHA`, would be good to note in the description that it's the first 7 letters